translator/pcconfig.py

```diff
@@ -0,0 +1,8 @@
+# The Pynecone configuration file.
```

Comment:
We recently updated the config format - could you update this?

You'll need to
1. delete this `pcconfig.py` file
2. run `pc init`with the latest version of Pynecone (pip 0.1.5 or the latest git). It should regenerate this file and look something like this:
```
import pynecone as pc

config = pc.Config(
    app_name="translator",
)
```
3. Running `pc init` after deleting the config file will replace your app code with the template app, you should be able to do `git checkout translator` to get the code back in the main folder.